src/kubernetes_backup/kubernetes_backup/s3.py

```diff
@@ -10,10 +12,34 @@ def __init__(self, bucket_name, prefix=''):
     self.bucket_name = bucket_name
     self.bucket_prefix = prefix
 
+  def __list_backups(self):
```

Comment:
Any reason why you're using double underscore here? It's usually only used when you don't want subclasses to implement said function themselves. Are you expecting code to subclass this object? If it's just meant to indicate internal use, use a single underscore (weak private function convention).

Reply:
i copied from [Substance repo](https://github.com/artsy/substance/blob/40445bb6b71906089ac508dc286a0853709a1524/lib/s3_interface.py#L14). agreed, single leading underscore is sufficient. [revised](https://github.com/artsy/opstools/pull/20/commits/2858bef8f386bcb770b37c80cb70d9af8be9aa60).